app/views/directives/osc-routing.html

```diff
@@ -170,29 +168,27 @@
     <div ng-show="secureRoute">
       <!-- TLS Termination -->
       <div class="form-group">
-        <label>TLS Termination</label>
-        <select ng-model="route.tls.termination" class="form-control">
-          <option value="edge">Edge</option>
-          <option value="passthrough">Passthrough</option>
-          <option value="reencrypt">Re-encrypt</option>
-        </select>
+        <label for="tlsTermination">TLS Termination</label>
+        <ui-select ng-model="route.tls.termination" input-id="tlsTermination" search-enabled="false">
+          <ui-select-match>{{$select.selected | humanizeTLSTermination}}</ui-select-match>
+          <ui-select-choices repeat="option in ['edge', 'passthrough', 'reencrypt']">
+            {{option | humanizeTLSTermination}}
+          </ui-select-choices>
+        </ui-select>
         <div class="learn-more-block help-block">
           <a href="{{'route-types' | helpLink}}" target="_blank">Learn More&nbsp;<i class="fa fa-external-link" aria-hidden="true"></i></a>
         </div>
       </div>
 
       <!-- Insecure Edge Termination Policy -->
       <div class="form-group">
-        <label>Insecure Traffic</label>
-        <select
-            ng-model="route.tls.insecureEdgeTerminationPolicy"
-            ng-disabled="route.tls.termination !== 'edge'"
-            class="form-control"
-            aria-describedby="route-insecure-policy-help">
-          <option value="">None</option>
-          <option value="Allow">Allow</option>
-          <option value="Redirect">Redirect</option>
-        </select>
+        <label for="insecureTraffic">Insecure Traffic</label>
+        <ui-select ng-model="route.tls.insecureEdgeTerminationPolicy" ng-disabled="route.tls.termination !== 'edge'" input-id="insecureTraffic" aria-describedby="route-insecure-policy-help" search-enabled="false">
```

Comment:
It looks like nothing is being preselected:

<img width="815" alt="openshift_web_console" src="https://cloud.githubusercontent.com/assets/1167259/21140403/d6942160-c105-11e6-9915-3eeba0e278c2.png">

It should default to `None`.